### PR TITLE
Navbar: Add `aria-label` attribute

### PR DESCRIPTION
**Pull request by austinoneil**
## Description

Add `aria-label` attribute to navbar.

References #2212

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

e2e tests, inspect element on storybook

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: stencil-workspace/src/components/modus-navbar/modus-navbar.tsx [hunk 1]
```diff
@@ -364,7 +367,7 @@ export class ModusNavbar {
 
     return (
       <Host id={this.componentId}>
-        <nav class={`${direction} ${shadow} ${variant}`}>
+        <nav class={`${direction} ${shadow} ${variant}`} aria-label={this.ariaLabel}>
```
Comment (coliff) [inline stencil-workspace/src/components/modus-navbar/modus-navbar.tsx:370]: If no aria-label is set then it renders as `<nav class aria-label>` - however it shouldn't show aria-label at all if not specified. Can you fix that please? By default it shouldn't have an aria-label at all.

**Review by coliff — CHANGES_REQUESTED** [review]

File: stencil-workspace/src/components/modus-navbar/modus-navbar.tsx (hunk 1 above)
Comment (austinoneil) [reply stencil-workspace/src/components/modus-navbar/modus-navbar.tsx:369]: I was not able to reproduce this locally.  When running locally, I got desired behavior.  I added some unit tests for these user journeys.

Comment (coliff) [reply stencil-workspace/src/components/modus-navbar/modus-navbar.tsx:369]: image

Adding `aria-label` to the modus-navbar is problematic, I think it should be on the nav only.

**Comment by austinoneil** [comment]
I changed the prop name to `navAriaLabel` and update documentation/tests.

**Comment by coliff** [comment]
It's still rendering empty aria-label if no aria-label is specified but it shouldn't. See code here:

image

**Comment by coliff** [comment]
Hi @austinoneil - the outstanding issue on this has been resolved now with your merged PR (empty aria-label fix) so once the conflicts are resolved here we can merge too. thanks :-)

**Comment by coliff** [comment]
doesn't seem to work:
https://deploy-preview-2379--moduswebcomponents.netlify.app/?path=/story/components-navbar--default&args=ariaLabel:default
This link should give the navbar an aria-label with default value but no value is given... its just rendered as `<nav class="" aria-label="">`

**Comment by austinoneil** [comment]
@coliff fixed.

**Review by coliff — APPROVED** [review]

**Pull request merged**